tasks/windows_client.yml

```diff
@@ -0,0 +1,102 @@
+---
+# tasks/windows_client.yml: Deploy various client-side configurations for Sensu-client on Windows
+# This contains all tasks, but these should be moved off into their respective places.
+
+    #- include_vars: "{{ ansible_distribution }}.yml"
+  - include_vars: "{{ ansible_os_family }}.yml"
+
+  - name: Download the Sensu installer msi into a local dir
+    win_get_url:
+      url: "{{ sensu_msi_download_path }}"
+      dest: "{{ sensu_win_temp_path }}\\sensu-installer.msi"
+
+    # If a version already exists on the target system, it should be made absent first.
```

Comment:
Hey @dkorel-copperleaf, could you explain this comment? Not sure I follow. 

Also for the task below, looking at http://docs.ansible.com/ansible/latest/win_package_module.html, you might be able to simply set `path: ` to the `sensu_msi_download_path` variable, looks like that actually accepts a URL natively. Did that not work?

Reply:
From my limited understanding of windows installer technology, I believe each new installation comes with a GUID which is stored into the registry. If subsequent installers honor the presence of this prior GUID they can upgrade that package - otherwise we end up with a brand new install, and potentially 2x versions of the same package... So i think here i was just ensuring we were installing the package we really wanted, and manually removing any others via some other method.